src/pkgx.sh/Hero.tsx

```diff
@@ -1,67 +1,127 @@
-import { Box, InputAdornment, Button, TextField, Typography, Stack, Snackbar, Alert, Tooltip, useMediaQuery, useTheme, Tabs, Tab } from "@mui/material";
-import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
-import ContentCopyIcon from '@mui/icons-material/ContentCopy';
-import React, { useState } from "react";
+import {
+	Box,
+	InputAdornment,
+	Button,
+	TextField,
+	Typography,
+	Stack,
+	Snackbar,
+	Alert,
+	Tooltip,
+	useMediaQuery,
+	useTheme,
+	Tabs,
+	Tab,
+} from "@mui/material"
+import ArrowForwardIcon from "@mui/icons-material/ArrowForward"
+import ContentCopyIcon from "@mui/icons-material/ContentCopy"
+import React, { useMemo, useState } from "react"
 import HeroTypography from "../components/HeroTypography"
+import { useSearchParams } from "react-router-dom"
 
 export default function Hero() {
-  const theme = useTheme();
-  const isxs = useMediaQuery(theme.breakpoints.down('md'));
+	const theme = useTheme()
```

Comment:
What’s going on with this indentation change?

Reply:
That's why I recommended a prettier and eslint config. Personally I use tabs instead of spaces, and other people that contribute will have different settings as well. It's a recurring issue that I've noticed in a lot of projects

Reply:
In Open Source the etiquette has long been to adopt the style choices of the project you contribute to